ext/pdo_mysql/mysql_driver.c

```diff
@@ -81,8 +81,8 @@ int _pdo_mysql_error(pdo_dbh_t *dbh, pdo_stmt_t *stmt, const char *file, int lin
 					dbh->is_persistent);
 			} else {
 				einfo->errmsg = pestrdup(
-					"Cannot execute queries while other unbuffered queries are active.  "
```

Comment:
I don't think there should be any whitespace changes in unrelated files.

Reply:
I agree with you, we can ignore this. I went up because they are really unnecessary (in my view).